html/includes/authentication/ldap.inc.php

```diff
@@ -100,7 +100,7 @@ function get_userlevel($username)
 {
     global $config, $ldap_connection;
 
-    $userlevel = 0;
+    $userlevel = -1;
```

Comment:
No specific reason. It was changed during my tests to rule out any mistakes. I can change it back to `0` if you want -- however, looking that how $userlevel is used, it doesn't matter if it's `0` or `-1`.